src/main/java/com/example/onlinebookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,20 @@
 package com.example.onlinebookstore.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
     private String title;
+    @NotNull
     private String author;
+    @NotNull
     private String isbn;
```

Comment:
ISBN consists from digits and it has [fixed length](https://uk.wikipedia.org/wiki/ISBN)

Reply:
Not resolved.